src/ConfigContext.ts

```diff
@@ -40,6 +40,7 @@ export interface ConfigContext {
   GITHUB_TEST_ENV_CLIENT_ID: string;
   GITHUB_CLIENT_SECRET?: string; // No need to inject secret for prod. Juno already knows it.
   isTerminalEnabled: boolean;
+  isPhoenixFlightDefault: boolean;
```

Comment:
Can we use a more descriptive variable name? Something like `isPhoenixEnabled`?

Reply:
Thought so but want to have flight name in the variable. Going with "isPhoenixEnabled"